sdk/core/core-client/review/core-client.api.md

```diff
@@ -223,13 +223,16 @@ export interface OperationRequestInfo {
 
 // @public
 export interface OperationRequestOptions {
+    accept?: string;
     allowInsecureConnection?: boolean;
+    contentType?: string;
```

Comment:
Maybe we should not allow customers set the accept and contentType by themselves in high level client and Modular ?

Reply:
Yeah I think we can remove these, I'm not sure what usage we needed them for before, but hopefully they are not necessary and can be overridden by setting the header explicitly if desired.